### DISCUSSION TITLE
Github Education

### DISCUSSION THREAD

**Discussion by Ritikkumar352** [community/community#108505] [GitHub Education]
### Select Topic Area

General

### Body

When I'm uploading my college Id card image and the clicking process my application, Nothing is happening after page loading.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Ritikkumar352 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by kalicsadam** [comment]
I have the same issue, and my yearly student pack expires soon.

**Reply by jeetamar28** [reply]
same here

**Comment by jeetamar28** [comment]
after successfully uploaded your college Id card image, just wait for 24-72 hr to verify
